kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -56,24 +56,46 @@ var ContentAreaView = BaseView.extend({
         this.$(".content").html("").append(view.$el);
     },
 
+    should_show_rating: function() {
+        /*
+        This function determines whether a rating should be shown for the content item.
+        returns: true or false
+        */
+        var entry_available = (typeof this.model !== "undefined") && !!this.model.get("available");
+        var logged_in = typeof window.statusModel.get("user_id") !== "undefined";
+        return logged_in && entry_available;
+    },
+
+    remove_rating_view: function() {
+        // Remove the rating view if it exists. Uses rating_view.close, so that the container element is not removed
```

Comment:
It sounds like you want to keep the DOM element around, but don't care so much about the view.

I would recommend appending the view.el to the DOM element in question, therefore, rather than setting it as the 'el' property on instantiation. You will just end up with an extra nested div, which should make no difference.